app/api/thesauri/specs/thesauri.spec.js

```diff
@@ -277,6 +277,21 @@ describe('thesauri', () => {
           })
         );
       });
+
+      describe('when the name of thesaurus is updated', () => {
+        it('should update the translations', async () => {
+          const data = { ...fixtures.dictionaries[1], name: 'new name' };
+          const response = await thesauri.save(data);
+          expect(translations.updateContext).toHaveBeenCalledWith(
+            response._id,
+            'new name',
+            { 'dictionary 2': 'new name' },
+            [],
+            { 'new name': 'new name', 'value 1': 'value 1', 'value 2': 'value 2' },
+            'Dictionary'
+          );
```

Comment:
@mfacar pls check that the values are actually on the db instead of the toHaveBeenCalled, this kind of mocking has proven worse in the past when changes on the system happens, providing with false positives.

Reply:
Done! 
I had to move the test because the old describe was using the translations as spy functions.